src/prepack-options.js

```diff
@@ -91,3 +94,14 @@ export function getSerializerOptions({
   if (additionalFunctions) result.additionalFunctions = additionalFunctions;
   return result;
 }
+
+export function getDebuggerOptions({
+  debugInFilePath = "./src/debugger/.sessionlogs/proxy2debugger.txt",
```

Comment:
This file name does not suggest that this is a communication "pipe" between the prepack debug engine and the adapter.